packages/zapp/console/src/components/common/Icons/RerunIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { IconProps } from './interface';
```

Comment:
Sorry,  I asked you to put RerunIcon as Svg, but I forgot that we curently have 2 InfoIcons in the system.
To properly support plugin system in future we need to decouple our current "monolith" (highly intertwined) application into packages. In this system all icons should leave in one layer: Inside `flyteconsole/composites/ui-atoms/src/Icons`. In future all Icons we currently have would be shown in Storybook.

Can you please rename this file to index.tsx and  move it to `flyteconsole/composites/ui-atoms/src/Icons/RerunIcon`
Then you will need to add `export { InfoIcon } from './InfoIcon';` inside `src/Icons/index.tsx`
then import it as : `import { RerunIcon } from '@flyteconsole/ui-atoms';`

Reply:
fixed